### ISSUE TITLE
The chainId in the returned data is empty

### ISSUE THREAD

**Issue by VliceZ** [WalletConnect/kotlin-walletconnect-lib#30]
How can i get it right？

**Comment by montaigne2100** [comment]
@VliceZ  Hi, did you manage to make it work?

**Comment by VliceZ** [comment]
i have no idea，just shows a dialog for users to choose，like ETH、BSC。

**Comment by ligi** [comment]
What exactly is the problem with this lib?
The lib should show no dialog.

**Comment by VliceZ** [comment]
What I mean is that I don’t know the chainId returned, so I let the user choose which chain the specific wallet is.

**Comment by montaigne2100** [comment]
@VliceZ  @ligi The sample app is not working and when I fix it and click to connect button, it navigates me to the wallet app I chose and returns me this strings:   ![isthis](https://user-images.githubusercontent.com/77534717/109544567-c87b9900-7ad8-11eb-8135-459e98115da2.PNG) It just oppens the app not asks for permission. How can  I fix it?

**Comment by VliceZ** [comment]
The bridge parameter in in simple app is not working, change it to "https://bridge.walletconnect.org"。

**Comment by montaigne2100** [comment]
@VliceZ Hi, thank you so much for posting this code snippet :) I am now able to send permission request but it crashes when user accepts connection. Error: 
E/AndroidRuntime: FATAL EXCEPTION: OkHttp https://bridge.walletconnect.org/...
    Process: com.application.kotlinwalletconnectdeneme, PID: 20277
    java.lang.IllegalStateException: Module with the Main dispatcher had failed to initialize
    
Can you take a look at this please? Thank you so much :)

**Comment by VliceZ** [comment]
@montaigne2100  I think the error is  from  this line "icons.add("https://images/xx.png")"，the icons url just a example , you need add a real icon Url.

**Comment by montaigne2100** [comment]
@VliceZ Yeah I did that but problem was about coroutines library's version. I upgraded it and its fixed. Thank you so so much :) You saved my days

**Comment by ligi** [comment]
So the problem is solved by using a valid icon URL? Maybe we should throw an exception as the library then

**Comment by dophunhan** [comment]
How to get chaninId @ligi @VliceZ @kashifahmad662 @montaigne2100 
I need to get the channel ID to check if the network matches or not
